packages/server/src/routes/sessions.js

```diff
@@ -14,14 +15,14 @@ const {
 } = require('../db');
 const { isUSDRSuperAdmin } = require('../lib/access-helpers');
 
-// NOTE(mbroussard): previously we allowed 2 uses to accommodate automated email systems that prefetch
-// links. Now, we send login links through a clientside redirect instead so this should not be necessary.
-const MAX_ACCESS_TOKEN_USES = 1;
+// Increasing the max-uses to ensure users are able to log-in even if their email client/security provider has clicked on the link already.
+// Specifically the issue was identified with Microsoft Safe Links, which clicks on the link to check if it is safe.
+const MAX_ACCESS_TOKEN_USES = 4;
```

Comment:
@as1729 I'm a bit late to the party, but I'm curious why this is necessary alongside the `isMicrosoftSafeLinksRequest` middleware. If it's not, I think a follow-up PR is warranted to turn this back down.

My reading of the middleware is that it will always emit a 200 status response and stop handling the request before the actual router handler function is called, and therefore the token won't have actually be used to authenticate a session. If that's accurate, then it doesn't seem like increasing the `MAX_ACCESS_TOKEN_USES` limit is necessary. Preventing token reuse (for actually authenticating a session) would be much preferred from a security standpoint.

In other words, it seems like the route should be accessible without any limit as long as `isMicrosoftSafeLinksRequest` returns early, because we're not actually establishing a new session for the user. But once a magic link token has authenticated a user, it should not be able to do so a second time.